UI/MainForm.cs

```diff
@@ -67,10 +68,11 @@ private void InitializeUILanguage()
 
         public event EventHandler<ConfigFile> ConfigLoaded;
 
+        private readonly LogAppenderFile logAppenderFile = new LogAppenderFile();
```

Comment:
This moved out of InitializeLogging() so it's available in the menu click handler.